pkg/daemon/ceph/osd/daemon.go

```diff
@@ -223,13 +229,21 @@ func getAvailableDevices(context *clusterd.Context, desiredDevices []DesiredDevi
 
 			if err == nil && matched {
 				// the current device matches the user specifies filter/list, use it for data
-				available.Entries[device.Name] = &DeviceOsdIDEntry{Data: unassignedOSDID, Config: matchedDevice}
+				deviceInfo = &DeviceOsdIDEntry{Data: unassignedOSDID, Config: matchedDevice}
 			} else {
 				logger.Infof("skipping device %s that does not match the device filter/list (%v). %+v", device.Name, desiredDevices, err)
 			}
 		} else {
 			logger.Infof("skipping device %s until the admin specifies it can be used by an osd", device.Name)
 		}
+
+		if deviceInfo != nil {
+			if partCount > 0 {
+				deviceInfo.LegacyPartitionsFound = ownPartitions
+			}
+
```

Comment:
Is this space necessary/helpful? I don't have a strong preference; just observing.

Reply:
Since `goreturns` doesn't have an opinion, i don't really either. Sometimes i add a space and other times not.